src/include/sfptpd_general_config.h

```diff
@@ -38,6 +38,7 @@
 #define SFPTPD_DEFAULT_DISCIPLINE_ALL_CLOCKS       (true)
 #define SFPTPD_DEFAULT_NON_SFC_NICS                (false)
 #define SFPTPD_DEFAULT_ASSUME_ONE_PHC_PER_NIC      (false)
+#define SFPTPD_DEFAULT_PHC_DEDUP                   (false)
```

Comment:
Given that this adjusts clocks, agree that this should be something customers have to request.